pages/common/gprof.md

```diff
@@ -4,14 +4,18 @@
 > It profiles the function executions of a program.
 > More information: <https://ftp.gnu.org/old-gnu/Manuals/gprof-2.9.1/html_mono/gprof.html>.
 
-- Compile binary with gprof information and run it to get `gmon.out`:
+- Compile binary to default `a.out` with gprof information and run it to get `gmon.out`:
 
 `gcc -pg {{program.c}} && {{./a.out}}`
```

Comment:
```suggestion
`gcc -pg {{program.c}} && ./a.out`
```

I think that this should not be a placeholder because it cannot be freely chosen by the user.